src/shared/xxAMIDOxx.xxSTACKSxx.Shared.Messaging.AWS.SNS/AwsSnsConfiguration.cs

```diff
@@ -4,5 +4,5 @@ namespace xxAMIDOxx.xxSTACKSxx.Shared.Messaging.AWS.SNS;
 
 public class AwsSnsConfiguration
 {
-    public Secret TopicArn { get; init; }
+    public Secret? TopicArn { get; init; }
 }
```

Comment:
Beeline at end of file missing (I know you didn't originally do that)

Reply:
👍